auslib/admin/views/scheduled_changes.py

```diff
@@ -0,0 +1,214 @@
+import json
+
+from sqlalchemy.sql.expression import null
+
+from flask import jsonify, request, Response, make_response
+
+from auslib.admin.views.base import AdminView, HistoryAdminView
+from auslib.admin.views.forms import DbEditableForm
+
+
+class ScheduledChangesView(AdminView):
+    """/scheduled_changes/:namespace"""
+
+    def __init__(self, namespace, table):
+        self.namespace = namespace
+        self.path = "/scheduled_changes/%s" % namespace
+        self.table = table
+        self.sc_table = table.scheduled_changes
+        super(ScheduledChangesView, self).__init__()
+
+    def get(self):
+        if request.args.get("all"):
+            rows = self.sc_table.select()
+        else:
+            rows = self.sc_table.select(where={"complete": False})
+        ret = {"count": len(rows), "scheduled_changes": []}
+        for row in rows:
+            r = {}
+            # TODO: maybe we should require API consumers to use actual column names instead of rewriting?
+            # that may mean we can't re-use rules forms, but it makes this less ugly...
+            # if there's a way to make callers use the column names and still use rules forms we should definitely do that
```

Comment:
Yeah, I think this is probably the best way. Making life simpler for clients is usually better...